example/test.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
```

Comment:
I think this script should not be added here

Reply:
+1